Added relying parties that need signed assertions and fixed one deprecated property.

mar235av opened the pull request.

jprosser commented on conf/relying-party.xml:
```diff
@@ -97,7 +97,7 @@
 
         <!-- needing signed assertions -->
-        <bean parent="RelyingPartyByName" c:relyingPartyIds="#{ {'https://az1.qualtrics.com','https://saml.enplug.com','https://uweducation.communityforce.com','https://uweducationstaging.communityforce.com','https://login.apogeenet.net/sp','https://pvs.education.uw.edu','https://app.yodeck.com/api/v1/account/metadata/','https://sso.pitchbook.com/sso/saml/sp/metadata','https://uw.creatcom.washing ...
+        <bean parent="RelyingPartyByName" c:relyingPartyIds="#{ {'https://az1.qualtrics.com','https://saml.enplug.com','https://uweducation.communityforce.com','https://uweducationstaging.communityforce.com','https://login.apogeenet.net/sp','https://pvs.education.uw.edu','https://app.yodeck.com/api/v1/account/metadata/','https://sso.pitchbook.com/sso/saml/sp/metadata','https://uw.creatcom.washing ...
```
i see an extra ' in the line there

mar235av replied:
Oops. Fixed.

jprosser reviewed (approved):
Looks good, thanks!

mar235av commented:
I added three commits to this pull request:
Two commits added two more RPs that also need signed assertions.
While doing that, I noticed a warning for a deprecated property in the same file, so I fixed that by replacing the property name with its designated replacement per Shibboleth documentation.

jprosser reviewed (approved):
Looks good to my eye at least!
